crates/tauri-utils/src/config.rs

```diff
@@ -664,6 +664,11 @@ pub struct MacConfig {
   pub provider_short_name: Option<String>,
   /// Path to the entitlements file.
   pub entitlements: Option<String>,
+  /// Path to a Info.plist file to merge with the default Info.plist.
+  ///
+  /// Note that Tauri also looks for a `Info.plist` and `Info.macos.plist` file in the same directory as the Tauri configuration file.
```

Comment:
Afaik this is a lie, i don't think info.macos.plist is a thing yet (it should though). The PR itself works fine though.